src/main/java/nextstep/courses/domain/PaidSession.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.courses.domain;
+
+import nextstep.payments.domain.Payment;
+
+import java.time.LocalDate;
+
+public class PaidSession extends Session {
+    private int maxCapacity;
+    private Long tuitionFee;
```

Comment:
```suggestion
public class PaidSession extends Session {
    private int maxCapacity;
    private int tuitionFee;
```
유료강의이므로 수강료가 null일 수 없으니 래퍼클래스 대신 primitive 타입을 사용하는 것이 좋겠네요
또한 수강료가 long 타입만큼 필요한 범위인지도 고려해보면 좋겠습니다.
int 양수 표현 범위도 충분하다고 생각해요!